src/test/java/net/masterthought/cucumber/ReportGenerator.java

```diff
@@ -18,6 +18,7 @@
     public final static String JSON_DIRECTORY = "json/";
 
     protected static final String SAMPLE_JSON = "sample.json";
+    protected static final String SAMPLE_XML_JSON = "sample-xml.json";
```

Comment:
Sorry, given my solution is to modify the velocity template I don't see how I can test without an integration test.